androidApp/src/main/kotlin/com/intive/picover/profile/view/ProfileActions.kt

```diff
@@ -24,7 +23,7 @@ fun ProfileActions(onLogoutClick: () -> Unit, onDeleteAccountCLick: () -> Unit)
 	val items = listOf(
 		ProfileActionsItem.Licenses(
 			onClick = {
-				context.startActivity(Intent(context, OssLicensesMenuActivity::class.java))
+				LibsBuilder().start(context)
```

Comment:
Library has a multiplatform and compose support - to be migrated later

Reply:
nice!